run-system-tests/action.yml

```diff
@@ -434,6 +446,40 @@ runs:
           ./tests/api_tests/config
           ./${{ env.BRANCH_DIR }}/latest/api/index.html
 
+    - name: "Run S3 CLI System Tests"
+      shell: 'script --return --quiet --command "bash {0}"'
+      if: ${{ env.RUN_SUBSET_OF_TESTS == 'false' && env.RUN_S3_CLI_SYSTEM_TESTS == 'true' }}
+      run: |
+        echo
+        echo "======================================================"
+        echo "RUNNING S3 CLI SYSTEM TESTS:"
+        echo "======================================================"
+        echo
+
+        export HOME="/root"
+
+        GOPATH=$(echo $(go env GOPATH))
+
+        cd tests/cli_tests
+        system_tests_exit_code=0
+        echo "Sequential tests are currently running.... It will take a few minutes for initial test output to appear..."
+        echo
+        ${GOPATH}/bin/gotestsum --jsonfile test-output.json --hide-summary=output --format testname ${{ env.SHOULD_RETRY_FAILURES }} --raw-command -- ../../S3CLI_TEST_RUNNER_COMMAND.sh || system_tests_exit_code=$?
+        cat test-output.json | ${GOPATH}/bin/go-test-report --groupSize 1 --output ../../${{ env.BRANCH_DIR }}/${GITHUB_SHA}/cli/index.html --title "0Chain S3 CLI System tests [${{ env.BRANCH_DIR }}/${GITHUB_SHA:0:8}] ran against [${{ env.NETWORK_URL }}] at [${{ env.TEST_TIME }}]"
```

Comment:
This looks like it's overriding the CLI tests report page

Reply:
Fixed these, now it goes to /s3cli/ dir
![image](https://github.com/0chain/actions/assets/42718091/ffd383a7-51a8-49f0-a2fe-bb7f270d968a)